bin/turbo_rspec

```diff
@@ -54,23 +56,54 @@ OptionParser
     opts.on("--use-runtime-info", "Use runtime info for tests group splitting") do
       use_runtime_info = true
     end
+
+    opts.on("--enable-system-tests", "Run the system tests (defaults false)") do
+      enable_system_tests = true
+    end
+
+    opts.on("--disable-system-tests", "Don't run the system tests (defaults true)") do
```

Comment:
What's the advantage of allowing the `--disable-system-tests` flag? If it's the default, maybe we don't need a flag at all?

Reply:
I'm adding both so that I can change the default without breaking things. The sequence becomes:

1. This PR - add both options,
2. Add explicit flags to uses - the extra stderr lines may help here,
3. Switch the default,

Reply:
Ah, ok, if it's temporary, cool.